### PR TITLE
Redesigned Footer

### LINKED ISSUE #436: Redesign Footer to Include More Information about the Organization
## Description
Currently, the footer only displays the logo, which doesn't offer much information about the organization. Let's redesign the footer to provide more details.

#### Organization Logo
Continue displaying the organization's logo for brand recognition and visual appeal.

#### Data Repository Link
Direct link to the organization's data repository, where users can access datasets, resources, or related materials.

#### Flat Repository Explorer Link
Link to the flat repository explorer tool, allowing users to navigate and explore the organization's repositories efficiently.

#### Social Media Links
Links to the organization's social media profiles (e.g., Twitter, LinkedIn, GitHub) for users to stay updated and engaged.

#### Contact Information
Contact details such as email address or a contact form for users to reach out with inquiries .

#### Feedback Link
Github issue page link (Hardcoded) to get feedbacks (Link:- https://github.com/coronasafe/leaderboard/issues/new/choose)

### PR DESCRIPTION
**Pull request by mayank1365**
**Resolves #436**

## Description

This pull request updates the footer component to include:

- **Organization Logo**: Maintains brand recognition.
- **Data Repository Link**: Direct access to datasets and resources.
- **Flat Repository Explorer Link**: Efficient navigation of repositories.
- **Social Media Links**: Connects users with organizational profiles.
- **Contact Information**: Provides an email address or contact form for inquiries.

### Added environment variables to facilitate dynamic content in the footer.

- `NEXT_PUBLIC_FLAT_REPO_EXPLORER_URL`
- `NEXT_PUBLIC_DATA_SOURCE`
- `NEXT_PUBLIC_GITHUB_URL`
- `NEXT_PUBLIC_LINKEDIN_URL`
- `NEXT_PUBLIC_YOUTUBE_URL`
- `NEXT_PUBLIC_CONTACT_EMAIL`

## Testing

- Verified that new footer elements are displayed correctly.
- Ensured that links and contact information are functional.

### CODE REVIEW

**Comment by mayank1365** [comment]
@rithviknishad Please have a look at the PR.

File: package.json
Comment (rithviknishad) [inline package.json:1]: Could you discard the package version changes? Was it changed intentionally?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: package.json
Comment (mayank1365) [reply package.json:1]: I will resolve it; it might have happened by mistake.

Comment (mayank1365) [reply package.json:1]: please have a look i have discarded the changes.

File: pnpm-lock.yaml
Comment (rithviknishad) [inline pnpm-lock.yaml:1]: Could you discard the changes to the lockfile too

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml
Comment (mayank1365) [reply pnpm-lock.yaml:1]: please check now

**Comment by mayank1365** [comment]
@rithviknishad please have a look at  the resolved issues. Thanks

**Comment by mayank1365** [comment]
Please merge it if everything is okay. Thank you

**Comment by rithviknishad** [comment]
Not a fan of this design. Could you rethink with something better 😄 

image

File: components/Footer.tsx [hunk 1]
```diff
@@ -3,20 +3,60 @@ import { env } from "@/env.mjs";
...
+              alt={env.NEXT_PUBLIC_ORG_NAME as string}
+              width={140}
+              height={36}
+            />
+          </span>
+        </div>
+        <div className="mb-4">
+          <a
+            href={env.NEXT_PUBLIC_DATA_SOURCE}
+            className="mb-1 block font-bold text-blue-600 hover:underline dark:text-blue-400"
```
Comment (rithviknishad) [inline components/Footer.tsx:22]: Also, do not hard code specific colours. Different organizations would be having different theme configurations, and may not match with with theme.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by mayank1365** [comment]
> Not a fan of this design. Could you rethink with something better 😄
> 
> image

image


What do you think about this

File: components/Footer.tsx (hunk 1 above)
Comment (mayank1365) [reply components/Footer.tsx:22]: sure. I will take care of this

**Comment by mayank1365** [comment]
@rithviknishad please review it, and get it merged, Thanks.

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**